code/modules/mob/mob.dm

```diff
@@ -657,7 +664,7 @@ var/next_mob_id = 0
 			fall()
 		else if(ko || (!has_legs && !ignore_legs) || chokehold)
 			fall(forced = 1)
-	canmove = !(ko || resting || stunned || chokehold || buckled || (!has_legs && !ignore_legs && !has_arms))
+	canmove = !((is_nearcrit(src) ? 0 : ko) || resting || stunned || chokehold || buckled || (!has_legs && !ignore_legs && !has_arms))
```

Comment:
`ko` is a dumb variable that basically determines if you are KO on the floor or not, nearcrit should overwrite that